.github/workflows/build.yml

```diff
@@ -0,0 +1,47 @@
+name: build_push
+on:
+  push:
+    branches:
+      - stc-master
+# Environment variables available to all jobs and steps in this workflow
+env:
+  REGISTRY_NAME: k8scc01covidacr
+  CLUSTER_NAME: k8s-cancentral-02-covid-aks
+  CLUSTER_RESOURCE_GROUP: k8s-cancentral-01-covid-aks
```

Comment:
These should be parameterized using GitHub secrets. Is this approach coming from one of our other CI pipelines?

Reply:
I added these three variables as secrets.